suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx

```diff
@@ -28,12 +28,19 @@ export const ConnectingDeviceScreen = () => {
         <ConnectDeviceSreenView style={applyStyle(screenStyle)}>
             <VStack spacing="medium" alignItems="center">
                 <ActivityIndicator size="large" />
-                <Text variant="titleMedium">
-                    <Translation id="moduleConnectDevice.connectingDeviceScreen.title" />{' '}
+                <Box
+                    style={{
```

Comment:
You can pass these as props without using the style prop

Reply:
fixed in 450f5c103b